### ISSUE TITLE
NumberBox inline spin buttons are offset when description is used

### ISSUE THREAD

**Issue by geometrikal** [Kinnara/ModernWpf#23]
<img width="341" alt="Annotation 2020-01-14 122941" src="https://user-images.githubusercontent.com/1187528/72305959-9424fd80-36c9-11ea-886d-175760117fd7.png">

XAML:

```xml
<ui:NumberBox Header="Tray"
                          Description="0: rebus, 1: left tray, 2: right tray, 3: repeat"
                          Minimum="0"
                          Maximum="3"
                          SmallChange="1" 
                          SpinButtonPlacementMode="Inline"
                          Value="{Binding Tray}"/>
```

**Comment by Kinnara** [comment]
Good bug, thanks for reporting. It's not hard to fix but I'll first report it on WinUI as well to see how they are going to approach it. As a workaround you can do this:
```xaml
<StackPanel>
    <ui:NumberBox Header="Tray"
                  SpinButtonPlacementMode="Inline" />
    <TextBlock Text="0: rebus, 1: left tray, 2: right tray, 3: repeat"
               Foreground="{DynamicResource SystemControlDescriptionTextForegroundBrush}" />
</StackPanel>
```

**Comment by Kinnara** [comment]
Closed via 3d6b342469eb79ccc0ad20d3de65c3fac20c5d91.
